content/blog/2024-07-02-tmix-06/index.md

```diff
@@ -0,0 +1,66 @@
+
+<figure>
+
+<img src="Xilem Emoji Picker.png" style="height: auto" alt="Screenshot of an app, containing a 4x4 grid of labelled Emoji, zoom in and out buttons and pagination controls. 'cat face with wry smile' is selected" height="1132" width = "863">
+
+<figcaption>
+
+A toy Emoji viewer example ([xilem#420][]) running on Android.
+This example uses system fonts, so has limited platform support because of the caveat mentioned above.
+
+</figcaption>
+</figure>
+
+[xilem#310][] brought in some new ideas about how Xilem Core can be structured, which enables support for specialised kinds of `View`, such as for window properties.
+`Adapt` views built on this were used to create an [Elm](https://elm-lang.org/)-inspired example in [xilem#401][], showing the flexibility of the Xilem architecture.
+Philipp Mildenberger has been prolific in his work on Xilem, as a co-author of the new Xilem Core, and in adapting Xilem Web to use it.
+He also worked to bring support for `OneOf` views to Xilem Core in [xilem#394][] - these were previously only supported in Xilem Web.
+
+Masonry's observability support has also been improved, with automatic tracing to a temporary file in debug builds added in [xilem#384][].
+[xilem#396][] implements [rfcs#6][], changing the architecture of Masonry to make it easier to write new features and tests.                                                            
+
+There was also some excellent community engagement with Xilem in June.
+Veniamin Ilmer has created an open source [Minesweeper game](https://github.com/veniamin-ilmer/minesweeper_xilem/) using Xilem.
+The discussion on Zulip (in [#xilem > Minesweeper converted from Iced to Xilem](https://xi.zulipchat.com/#narrow/stream/354396-xilem/topic/Minesweeper.20converted.20from.20Iced.20to.20Xilem)) has been very useful in guiding areas for improvement.      
+
+<figure>
+
+<img style="height: auto" src="Minesweeper.png" alt="A window titled Minesweeper, with text in the top-left 'Mines: 82', a top-center button with a neutral ASCII emoticon, and a board represented by a grid of buttons 30 wide and 16 tall, where some buttons are replaced with colour labels containing digits, and some buttons contain an exclamation mark." height="589" width = "931">
+
+<figcaption>
+
+Screenshot by Veniamin Ilmer of their Minesweeper game, used under the [Apache 2.0](https://github.com/veniamin-ilmer/minesweeper_xilem/blob/main/LICENSE) license.
+
+</figcaption>
+</figure>
+
+Following on from May, Olivier wrote and published a [report](@/blog/2024-06-15-rustnl-2024-unconference.md) on our experience of the RustNL unconference.
+This led to some useful discussion online - see [the zulip thread](https://xi.zulipchat.com/#narrow/stream/181284-blogging/topic/Draft.20-.20Report.20on.20the.20RustNL.202024.20Conference/near/444974910) corresponding with that post for links.
+We are now in the final month of the [May-July 2024 roadmap](@/blog/2024-06-15-rustnl-2024-unconference.md).
```

Comment:
There doesn't seem to be much roadmap discussion on that page. Did you mean [2024-06-17-roadmap-may-2024.md](https://github.com/linebender/linebender.github.io/blob/main/content/blog/2024-06-17-roadmap-may-2024.md)?